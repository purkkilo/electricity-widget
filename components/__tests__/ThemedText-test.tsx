import { render } from "@testing-library/react-native";

import { ThemedText } from "../ThemedText";

describe("<ThemedText />", () => {
  test("Text renders correctly", () => {
    const tree = render(<ThemedText>Some text</ThemedText>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
