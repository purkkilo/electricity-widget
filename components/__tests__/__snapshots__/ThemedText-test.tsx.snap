// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ThemedText /> Text renders correctly 1`] = `
<Text
  style={
    [
      {
        "color": "#11181C",
      },
      {
        "fontSize": 16,
        "lineHeight": 24,
      },
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
    ]
  }
>
  Some text
</Text>
`;
